adagrams/game.py

```diff
@@ -1,13 +1,135 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+def create_pool_list():
+    pool_list = []
+    for letter, number_of_tiles in LETTER_POOL.items():
+        for index in range(0, number_of_tiles):
+            pool_list.append(letter)
+    return pool_list
+
+
 def draw_letters():
-    pass
+    pool_list = create_pool_list()
+    print(f'Letter pool: {pool_list}')
+    hand = []
+    letter_freq = {}
+    draw = True
+    while draw:
+        letter_index = randint(0, len(pool_list)-1) #Assigns the index of the letter based on random integer
```

Comment:
Take care to use white spaces correctly. We should surround operators like `=` with white spaces. Please review the PEP8 style guide on white spaces [here](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements) and take care to clean up your code before submission. In industry, pull requests with code that does not conform to code styles will need to be revised because we want the codebase to neat and consistent.
```suggestion
        letter_index = randint(0, len(pool_list) - 1) 
```